javascript-examples/README.md

```diff
@@ -0,0 +1,21 @@
+
+# OPTIMADE JavaScript Examples
+
+Learn how this works and create your own apps running everywhere:
+
+- [Load crystalline animations from MPDS via OPTIMADE](mpds_gifs.html)
+
+Compile the code locally on your own PC using the following Unix shell commands:
+
+```bash
+apt-get install nodejs # install Node JS here otherwise
```

Comment:
If you give instructions to install nodejs you should also give instructions for npm.